src/test/features/claimant-response/routes/payment-date.ts

```diff
@@ -158,7 +158,7 @@ describe('Claimant response: payment date', () => {
       context('when service is healthy', () => {
         const dataToSend = {
           date: {
-            year: 2019,
+            year: 2039,
```

Comment:
Please change to use dynamic date.

Reply:
It'll still be date-dependant either way. This date must be earlier than the defendant's proposed date (which is also fixed/static) and later than the court-calculated date, which is dynamic. I could make this dynamic as `now + 10years` perhaps